R/git.R

```diff
@@ -154,31 +154,51 @@ git_create_sample_log <- function(folder_path="/tmp"){
   git_repo <- file.path(folder_path,'.git')
 
   # git --git-dir sample/.git --work-tree sample add hello.R
+  #' @param git_repo The git repo path
+  #' @param folder_path The worktree path
+  #' @param file_to_add The file we want to add
+  #' @export
+  git_add <- function(git_repo, folder_path, file_to_add) {
```

Comment:
the function name is already git add, so lets just call the parameter `file`. Can the `filepath` be provided rather than just the file name? if so, then we should call it `filepath` instead!